common/TrikotFrameworkName.podspec

```diff
@@ -8,15 +8,17 @@ Pod::Spec.new do |spec|
     spec.summary                  = 'Awesome library'
 
     spec.static_framework         = true
-    spec.vendored_frameworks      = "build/bin/ios/TrikotFrameworkName.framework"
+    spec.vendored_frameworks      = "build/bin/TrikotFrameworkName.framework"
     spec.libraries                = "c++", "System"
     spec.module_name              = "#{spec.name}_umbrella"
 
     spec.pod_target_xcconfig = {
         'KOTLIN_BUILD_TYPE[config=Debug]' => 'DEBUG',
         'KOTLIN_BUILD_TYPE[config=Release]' => 'RELEASE',
         'KOTLIN_TARGET[sdk=iphonesimulator*]' => 'iosX64',
-        'KOTLIN_TARGET[sdk=iphoneos*]' => 'iosArm64'
+        'KOTLIN_TARGET[sdk=iphoneos*]' => 'iosArm64',
+        'KOTLIN_TARGET[sdk=appletvos*]' => 'tvosArm64',
+        'KOTLIN_TARGET[sdk=appletvsimulator*]' => 'tvosX64'
```

Comment:
Could we add watchOS as well in the eventually someone needs it?

Reply:
Should macOS and Catalyst be there too?
Does KPM support them?

Reply:
macOS is supported for sure but I don't think mac catalyst is as of now, at least not x86_64h.

Reply:
I guess that it should be something like this
```
'KOTLIN_TARGET[sdk=watchos*]' => 'watchosArm64',
'KOTLIN_TARGET[sdk=watchsimulator*]' => 'watchosX64'
```
but I don't know about if we need X32 and don't have the setup to make sure it work so I'll let the one who needs it contribute to trikot.patron with accurate values.

Reply:
Same for macOS

Reply:
👌